src/App.js

```diff
@@ -1,18 +1,23 @@
 import './App.css';
 import { Route, Routes } from 'react-router-dom';
-import { Cart, HomePage, MockAPI, ProductList, Wishlist, ProductDetails } from './pages/index';
+import { Cart, HomePage, MockAPI, ProductList, Wishlist, ProductDetails, PrivateRoute, SignUp, Login, UserProfile } from './pages/index';
 
 function App() {
   return (
     <div className="App">
       <Routes>
-        <Route path='/' element={<HomePage />}/>
+        <Route path="/" element={<HomePage />}/>
         <Route path="/productlist" element={<ProductList />} />
-        <Route path="/cart" element={<Cart />} />
-        <Route path="/wishlist" element={<Wishlist />} />
         <Route path="/productlist/:productId" element={<ProductDetails />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<SignUp />} />
         <Route path="/mock-api" element={<MockAPI />} />
-       </Routes>
+        <Route path="/user" element={<PrivateRoute />}>
```

Comment:
Good to see the usage of private route over here!

Reply:
Thanks